### ISSUE TITLE
Heads plugins

### ISSUE THREAD

**Issue by x1p** [Ruinscraft/Panilla#9]
When a player takes a head for one of a heads plugin (like Head-Database or EpicHeads), Panilla gives this warning:
`[Panilla] Packet to PlayerName123 (PacketPlayOutSetSlot) was dropped because NBT tag display failed checks`
The texture of the head is still the same everything seems ok. This is confirmed by looking at the nbt information.
```
[20:31:00] [Server thread/INFO]:  ->: SkullOwner (Compound)
[20:31:00] [Server thread/INFO]:   ->: Id (String): 04049c90-d3e9-4621-9caf-0000aaa13475
[20:31:00] [Server thread/INFO]:   ->: Properties (Compound)
[20:31:00] [Server thread/INFO]:    ->: textures (List): {Value:"eyJ0ZXh0dXJlcyI6eyJTS0lOIjp7InVybCI6Imh0dHA6Ly90ZXh0dXJlcy5taW5lY3JhZnQubmV0L3RleHR1cmUvZTE4YWUyMmVkMTU4ZjlhZGI5NDFiOWI2Y2ZmMTgwNzY3NGY3YjNiMjRmYjBkYzUzMWY5NWU0NmJlMDZiOGM5NCJ9fX0="}
[20:31:00] [Server thread/INFO]:  ->: display (Compound)
[20:31:00] [Server thread/INFO]:   ->: Name (String): {"text":"Apple"}
[20:31:00] [Server thread/INFO]:  ->: Damage (Int): 3
```
Used `strictness: average`.

Is there anything I can do to allow this without warnings or could you add compatibility?

**Comment by dextonanderson** [comment]
Hi,

It should now be fixed with this commit 1ce778daeb2f611e91900a24208dfc80d030e872

I have bumped the version and posted an update on SpigotMC.

Let me know if you still have this issue.

**Comment by x1p** [comment]
Seems to work just fine :ok_hand:
